benchadapt/python/benchadapt/_machine_info.py

```diff
@@ -77,6 +77,34 @@ class GitParseWarning(RuntimeWarning):
 
 
 def github_info():
+    """Get github metadata from environment variables"""
+    repo = os.environ.get("CONBENCH_PROJECT_REPOSITORY")
+    pr_number = os.environ.get("CONBENCH_PROJECT_PR_NUMBER") or os.environ.get(
+        "BENCHMARKABLE_PR_NUMBER"
+    )
+    commit = os.environ.get("CONBENCH_PROJECT_COMMIT")
+
+    gh_info = {
+        "commit": commit,
+        "repository": repo,
+        # "branch": None,
+        "pr_number": pr_number,
+    }
+
+    if not (repo and commit):
+        warnings.warn(
+            (
+                "Both CONBENCH_PROJECT_REPOSITORY and CONBENCH_PROJECT_COMMIT must be "
+                "set if `github` is not specified. CONBENCH_PROJECT_PR_NUMBER should be "
+                "null for builds on the default branch."
```

Comment:
Very minor, and very pedantic: when you say "CONBENCH_PROJECT_PR_NUMBER should be null" does that mean unset or `""`?

Reply:
Digging through, I'm fairly sure that because [this line](https://github.com/conbench/conbench/blob/0efd2b9132c73034572e13ea74527b24ecb12288/conbench/entities/commit.py#L499) and both `not None` and `not ""` will return `True`, it doesn't matter. But I'll change this to recommend not setting it so nobody tries quoted `"null"` or anything else silly